src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,16 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+
+  &__card {
+    width: $cardWidth;
+
+    &:not(:nth-child(#{$cardsInRow}n)) {
+      margin-right: $horizonMarginBetweenCards;
```

Comment:
u use this var in a single place, so.. let's move it to this file0

Reply:
but I also use them in main.scss